packages/genesys-spark-components/src/components/beta/gux-option-status/gux-option-status.types.ts

```diff
@@ -0,0 +1,5 @@
+export type GuxStatusIndicatorVariant =
+  | 'success'
+  | 'info'
+  | 'warning'
+  | 'error';
```

Comment:
It is probably correct but my first thought was this is related to the badge component and the badge component has an additional accent of `inherit`. Would that accent be useful here too or not needed?